site/index.html

```diff
@@ -76,6 +76,54 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Rapid prototyping</h3>
+            <p>Start small and explore new ideas to find your market fit. Haskell is a compact language with a large ecosystem of open-source libraries and tools
+               to get you started and help you grow.
+               The strong type system makes it easy to write programs that work, both large and small, even when you don't like writing types&mdash;the compiler will figure it out.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Scale with ease</h3>
+            <p>From 1 to 100k users and beyond. Haskell offers best-in-class support for async, concurrent and parallel programming.
+               It can take advantage of multi-core processors with ease thanks to garbage collection, green threads and immutability everywhere.
+               Additionally, you can use advanced streaming libraries for efficient data processing
+               &mdash; all while keeping complexity low as your user base and code base grow.
```

Comment:
I think I must have missed this one.

```suggestion
               Additionally, you can use advanced streaming libraries for efficient data processing&mdash;all
               while keeping complexity low as your user base and code base grow.
```

Reply:
(probably did the duplicated one by mistake instead of this)